VSCodium (1.70.2, undefined, desktop)
Jupyter Extension Version: 2022.8.1002170547.
Python Extension Version: 2022.14.0.
Multiple Workspace folders opened /home/eragon/repos/fluves/pyiotdataflow, /home/eragon/repos/fluves/iot-modeling, /home/eragon/repos/fluves/dali-intrusion-data-analysis, /home/eragon/repos/fluves/infrastructure, /home/eragon/repos/fluves/cn-ws, /home/eragon/repos/cheatsheet, /home/eragon/repos/fluves/pydts, /home/eragon/repos/fluves/pycnws
info 9:29:11.84: ZMQ install verified.
info 9:29:11.412: LSP Notebooks experiment is disabled -- Pylance disabled or not installed
info 9:29:11.433: Experiment status for python is {"enabled":true,"optInto":[],"optOutFrom":[]}
info 9:29:11.479: Preferred Remote kernel for /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb is undefined
info 9:29:11.480: Find preferred kernel for /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb with metadata {"kernelspec":{"display_name":"Python 3.7.13 ('pyiotdataflow')","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.7.13"},"orig_nbformat":4,"vscode":{"interpreter":{"hash":"9a5e8fbb23ceec6fe44410afae2723d876dde8a59a88179184ea183507ad9ed0"}}} & preferred interpreter /bin/python
info 9:29:11.483: Preferred Remote kernel for /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb is undefined
info 9:29:11.484: Preferred kernel .jvsc74a57bd09a5e8fbb23ceec6fe44410afae2723d876dde8a59a88179184ea183507ad9ed0./opt/conda/envs/pyiotdataflow/python./opt/conda/envs/pyiotdataflow/python.-m#ipykernel_launcher is exact match or top match for non python kernels, (false, undefined, true, false)
info 9:29:11.484: PreferredConnection: .jvsc74a57bd09a5e8fbb23ceec6fe44410afae2723d876dde8a59a88179184ea183507ad9ed0./opt/conda/envs/pyiotdataflow/python./opt/conda/envs/pyiotdataflow/python.-m#ipykernel_launcher found for NotebookDocument: /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:11.509: Starting Jupyter Session id = 'startUsingPythonInterpreter:.jvsc74a57bd09a5e8fbb23ceec6fe44410afae2723d876dde8a59a88179184ea183507ad9ed0./opt/conda/envs/pyiotdataflow/python./opt/conda/envs/pyiotdataflow/python.-m#ipykernel_launcher' (Python Path: /opt/conda/envs/pyiotdataflow, EnvType: Conda, EnvName: 'pyiotdataflow', Version: 3.7.13) for '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb' (disableUI=true)
info 9:29:11.510: Creating raw notebook for resource '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb'
info 9:29:11.510: Computing working directory for resource '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb'
info 9:29:11.568: Starting raw kernel 'pyiotdataflow (Python 3.7.13)' for interpreter /opt/conda/envs/pyiotdataflow/bin/python
info 9:29:11.577: Kernel launching with ports 9000,9001,9002,9003,9004. Start port is 9000
info 9:29:11.786: Process Execution: > /bin/python -m pip list
> /bin/python -m pip list
info 9:29:11.819: Process Execution: > /bin/python -c "import pip"
> /bin/python -c "import pip"
error 9:29:11.820: Failed to get activated environment variables ourselves [Error: Command failed: conda activate pyiotdataflow && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/eragon/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printEnvVariables.py

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.



	at ChildProcess.exithandler (node:child_process:406:12)
	at ChildProcess.emit (node:events:390:28)
	at maybeClose (node:internal/child_process:1064:16)
	at Socket.<anonymous> (node:internal/child_process:450:11)
	at Socket.emit (node:events:390:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 1,
  signal: null,
  cmd: "conda activate pyiotdataflow && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/eragon/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printEnvVariables.py"
}
error 9:29:11.821: Execing shell command, Class name = g, completed in 328ms, has a falsy return value, Arg 1: "~/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printEnvVariables.py", Arg 2: "" [Error: Command failed: conda activate pyiotdataflow && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/eragon/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printEnvVariables.py

CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.



	at ChildProcess.exithandler (node:child_process:406:12)
	at ChildProcess.emit (node:events:390:28)
	at maybeClose (node:internal/child_process:1064:16)
	at Socket.<anonymous> (node:internal/child_process:450:11)
	at Socket.emit (node:events:390:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 1,
  signal: null,
  cmd: "conda activate pyiotdataflow && echo 'e8b39361-0157-4923-80e1-22d70d46dee6' && python /home/eragon/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printEnvVariables.py"
}
info 9:29:12.180: Preferred Remote kernel for /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb is undefined
info 9:29:13.27: Process Execution: > /opt/conda/envs/pyiotdataflow/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"9d3dcaf1-857d-45ef-8c0e-76bb4885b269" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/eragon/.local/share/jupyter/runtime/kernel-v2-6012kH0EIgrniHzA.json
> /opt/conda/envs/pyiotdataflow/bin/python -m ipykernel_launcher --ip=127.0.0.1 --stdin=9003 --control=9001 --hb=9000 --Session.signature_scheme="hmac-sha256" --Session.key=b"9d3dcaf1-857d-45ef-8c0e-76bb4885b269" --shell=9002 --transport="tcp" --iopub=9004 --f=/home/eragon/.local/share/jupyter/runtime/kernel-v2-6012kH0EIgrniHzA.json
info 9:29:13.27: Process Execution: cwd: ~/repos/fluves/pyiotdataflow/notebooks
cwd: ~/repos/fluves/pyiotdataflow/notebooks
info 9:29:13.49: Process Execution: > /opt/conda/envs/pyiotdataflow/bin/python -m pip list
> /opt/conda/envs/pyiotdataflow/bin/python -m pip list
info 9:29:13.104: Process Execution: > /opt/conda/envs/pyiotdataflow/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
> /opt/conda/envs/pyiotdataflow/bin/python -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
info 9:29:13.212: ipykernel version 6.15.2 for /opt/conda/envs/pyiotdataflow/bin/python
info 9:29:13.212: ipykernel location /opt/conda/envs/pyiotdataflow/lib/python3.7/site-packages/ipykernel/__init__.py for /opt/conda/envs/pyiotdataflow/bin/python
warn 9:29:13.501: StdErr from Kernel Process /opt/conda/envs/pyiotdataflow/lib/python3.7/site-packages/traitlets/traitlets.py:2395: FutureWarning: Supporting extra quotes around strings is deprecated in traitlets 5.0. You can use 'hmac-sha256' instead of '"hmac-sha256"' if you require traitlets >=5.
  FutureWarning,
/opt/conda/envs/pyiotdataflow/lib/python3.7/site-packages/traitlets/traitlets.py:2349: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use '9d3dcaf1-857d-45ef-8c0e-76bb4885b269' instead of 'b"9d3dcaf1-857d-45ef-8c0e-76bb4885b269"'.
  FutureWarning,

info 9:29:13.508: Kernel Output: NOTE: When using the `ipython kernel` entry point, Ctrl-C will not work.

To exit, you will have to explicitly quit this process, by either sending
"quit" from a client, or using Ctrl-\ in UNIX-like environments.

To read more about this, see https://github.com/ipython/ipython/issues/2049


To connect another client to this kernel, use:
    --existing kernel-v2-6012kH0EIgrniHzA.json

info 9:29:13.656: Started kernel pyiotdataflow (Python 3.7.13)., (Raw session started and connected)
info 9:29:13.656: Finished connecting da8ec01b-e289-48a2-b06b-7ee9f213fb05
info 9:29:13.658: UpdateWorkingDirectoryAndPath in Kernel
info 9:29:13.658: Executing silently Code (idle) = import os\nimport sys\n%cd "//home/eragon/repos/fluves/pyiotdataflow/notebooks"\nif os.getcwd() not in
info 9:29:14.31: Executing silently Code (completed) = import os\nimport sys\n%cd "//home/eragon/repos/fluves/pyiotdataflow/notebooks"\nif os.getcwd() not in with 1 output(s)
info 9:29:14.31: Waiting for idle on (kernel): 2125cd2a-f59a-4f9f-ade6-e025727d959f -> idle
info 9:29:14.31: Finished waiting for idle on (kernel): 2125cd2a-f59a-4f9f-ade6-e025727d959f -> idle
info 9:29:14.40: Process Execution: > /opt/conda/envs/pyiotdataflow/bin/python ~/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printJupyterDataDir.py
> /opt/conda/envs/pyiotdataflow/bin/python ~/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/pythonFiles/printJupyterDataDir.py
info 9:29:14.42: Widget Message: Received IPyWidgets_kernelOptions in ScriptManager
info 9:29:14.42: Widget Message: Received IPyWidgetMessages.IPyWidgets_kernelOptions
info 9:29:14.54: Widget Message: data-base-url set to https://file%2B.vscode-resource.vscode-cdn.net/home/eragon/.vscode-oss/extensions/ms-toolsai.jupyter-2022.8.1002170547/tmp/scripts/5cf26933856265a9cecc44aab87d6c7bea72d91df64b33c65c30eca622d0492d/jupyter
info 9:29:15.491: Execute Cell 2 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 4 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 5 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 6 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 8 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 9 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 11 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 12 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:15.492: Execute Cell 13 /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb
info 9:29:16.245: Cell 2 executed with state Success
info 9:29:16.287: Cell 4 executed with state Success
info 9:29:35.725: Cell 5 executed with state Success
info 9:29:35.773: Cell 6 executed with state Success
info 9:29:35.823: Cell 8 executed with state Success
info 9:29:35.869: Cell 9 executed with state Success
info 9:29:35.920: Cell 11 executed with state Success
info 9:29:36.482: Cell 12 executed with state Success
info 9:29:36.636: Cell 13 executed with state Success
info 10:12:25.330: Disposing kernel .jvsc74a57bd09a5e8fbb23ceec6fe44410afae2723d876dde8a59a88179184ea183507ad9ed0./opt/conda/envs/pyiotdataflow/python./opt/conda/envs/pyiotdataflow/python.-m#ipykernel_launcher for notebook /home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb due to selection of another kernel or closing of the notebook
info 10:12:25.330: Dispose Kernel '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb' associated with '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb'
info 10:12:25.339: Cancel pending cells
info 10:51:2.291: Shutting down notebooks for da8ec01b-e289-48a2-b06b-7ee9f213fb05
info 10:51:2.291: Disposing HostJupyterExecution 0005baa3-fd68-45fa-9ed6-77db964aa561
info 10:51:2.291: Disposing super HostJupyterExecution 0005baa3-fd68-45fa-9ed6-77db964aa561
info 10:51:2.291: Dispose Kernel '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb' associated with '/home/eragon/repos/fluves/pyiotdataflow/notebooks/hydromet.ipynb'
info 10:51:2.291: Disposing HostJupyterExecution 0005baa3-fd68-45fa-9ed6-77db964aa561
info 10:51:2.291: Finished disposing HostJupyterExecution  0005baa3-fd68-45fa-9ed6-77db964aa561
info 10:51:2.300: Cleaning up server cache 0005baa3-fd68-45fa-9ed6-77db964aa561
info 10:51:2.300: Finished disposing HostJupyterExecution  0005baa3-fd68-45fa-9ed6-77db964aa561
